### DISCUSSION TITLE
Docusaurus Plugin / Setup

### DISCUSSION THREAD

**Discussion by ddresch** [avitorio/outstatic#98] [Ideas]
At the moment there is no WYSIWYG editor plugin available (afaik) for docusaurus, which is as suitable like outstatic. Right now outstatic is totally nextJS dependent, right? 

If it's possible to use it also for netlify and in an non nextjs environment like docusaurus it could easily be a replacement for gitbook or other services like that.

**Comment by avitorio** [comment]
Hi @ddresch,

Thanks for the suggestion. You're correct, Outstatic is totally Next.js dependent. It is basically a Next.js application, but I think it could be used to edit the .md for other projects just by using the correct branch and path on the .env. You'd still need to have a Next.js install for Outstatic hosted somewhere.

**Comment by ddresch** [comment]
Thanks for your feedback. I stumbled across decap CMS (netlify cms) yesterday, which is doing exactly what I wanted to achieve with outstatic. I guess you know the project: https://decapcms.org/

**Reply by avitorio** [reply]
Hey @ddresch, I was aware of Netlify CMS but not of Decap. Happy to know it serves your purpose. Good luck! :)
